### PR TITLE
feat: new component

### PR DESCRIPTION
**Pull request by xiaaa123**
## Description

Please provide a brief summary of the changes. If it is related to an issue, please bring the issue number, like `fixes #${number}`.

## Package

- [ ] tuya-panel-animation-sdk
- [ ] tuya-panel-api
- [ ] tuya-panel-electrician-sdk
- [✅] tuya-panel-ipc-sdk
- [ ] tuya-panel-lamp-sdk
- [ ] tuya-panel-lock-sdk
- [ ] tuya-panel-robot-sdk

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [✅] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: example/src/scenes/Ipc/index.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import IpcPlayer from './IpcPlayer';
 import IpcBatteryScene from './IpcBattery';
 import IpcPlayerRnPageTest from './IpcPlayerRnPageTest';
 import IpcVideoBit from './IpcVideoBit'
```
Comment (youngjuning) [inline example/src/scenes/Ipc/index.js:12]:
```suggestion
    import IpcVideoBit from './IpcVideoBit';
```

File: example/src/scenes/Ipc/index.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import IpcPlayer from './IpcPlayer';
...
+import IpcPtz from './IpcPtz'
```
Comment (youngjuning) [inline example/src/scenes/Ipc/index.js:13]:
```suggestion
import IpcPtz from './IpcPtz';
```

File: example/src/scenes/Ipc/index.js [hunk 3]
```diff
@@ -23,6 +24,7 @@ export default class AnimatedScene extends Component {
   static IpcBattery = IpcBatteryScene;
   static IpcPlayerRnPageTest = IpcPlayerRnPageTest;
   static IpcVideoBit = IpcVideoBit;
+  static IpcPtz = IpcPtz
```
Comment (youngjuning) [inline example/src/scenes/Ipc/index.js:27]:
```suggestion
  static IpcPtz = IpcPtz;
```

**Review by youngjuning — APPROVED** [review]
LGTM~

**Comment by xiaaa123** [comment]
okey, thank you very much !

**Pull request merged**